src/Mod/Sketcher/Gui/DrawSketchHandlerLine.h

```diff
@@ -306,11 +306,17 @@ void DSHLineController::configureToolWidget()
     onViewParameters[OnViewParameter::Second]->setLabelType(Gui::SoDatumLabel::DISTANCEY);
 
     if (handler->constructionMethod() == ConstructionMethod::OnePointLengthAngle) {
-        onViewParameters[OnViewParameter::Fourth]->setLabelType(Gui::SoDatumLabel::ANGLE);
+        onViewParameters[OnViewParameter::Fourth]->setLabelType(
+            Gui::SoDatumLabel::ANGLE,
+            Gui::EditableDatumLabel::Function::Dimensioning);
     }
     else {

```

Comment:
Also here it is problematic too. Because the else is for the 2 other methodes : 
- 1 point + width / height. For which you want the labels to be 'Dimensioning'.
- 2 Points : for this methode, the 4 labels are positioning because you are positioning 2 points. So for this methode third and fourth need to be positioning. It is to be noted that this methode is useless if the positioning OVP are hidden.